tests/test_models.py

```diff
@@ -7,6 +7,7 @@
 from timeless.roles.models import Role
 from timeless.schemetypes.models import SchemeType
 from timeless.items.models import Item
+from timeless.items.models import ItemHistory
```

Comment:
@ammaratef45 pls do it in this way `from timeless.items.models import Item, ItemHistory `